src/active-response/active_responses.h

```diff
@@ -87,6 +87,13 @@ const cJSON* get_alert_from_json(const cJSON *input);
  * */
 const char* get_srcip_from_json(const cJSON *input);
 
+/**
+ * Get srcip from win eventdata
+ * @param data Input
+ * @return cJSON * with the ipAddress or NULL on fail
+ * */
+cJSON* get_srcip_from_win_eventdata(const cJSON *data);
```

Comment:
This function could be static and located in the `.c` file.

Reply:
Done!